tests/test_encryption.py

```
@@ -261,3 +261,114 @@ def test_each_redacted_key_is_removed(self, key: str) -> None:
+            "additional_repos": [
+                {
+                    "name": "infra",
+                    "url": "https://github.com/org/infra",
+                    "token": "tok",
+                },  # pragma: allowlist secret
+            ],
+        }
+        encrypted = encrypt_sensitive_fields(params)
+        decrypted = decrypt_sensitive_fields(encrypted)
+        assert (
+            decrypted["additional_repos"][0]["token"] == "tok"
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to: "token"

(S105)